aslprep/utils/atlas.py

```diff
@@ -1,33 +1,45 @@
 """Functions for working with atlases."""
 
 
-def get_atlas_names():
+def get_atlas_names(subset):
     """Get a list of atlases to be used for parcellation and functional connectivity analyses.
 
     The actual list of files for the atlases is loaded from a different function.
 
     NOTE: This is a Node function.
 
+    Parameters
+    ----------
+    subset = {"all", "subcortical", "cortical"}
+        Description of the subset of atlases to collect.
+
     Returns
     -------
     :obj:`list` of :obj:`str`
         List of atlases.
     """
-    return [
-        "Schaefer117",
-        "Schaefer217",
-        "Schaefer317",
-        "Schaefer417",
-        "Schaefer517",
-        "Schaefer617",
-        "Schaefer717",
-        "Schaefer817",
-        "Schaefer917",
-        "Schaefer1017",
-        "Glasser",
-        "Gordon",
-        "Tian",
-    ]
+    atlases = {
+        "cortical": [
+            "4S152Parcels",
```

Comment:
these are technically both. Is it worth getting to that level of detail?

Reply:
The distinction is really "is there cortex" vs. "no cortex" for the surface parcellations in XCP-D (e.g., we can't produce parcellated cortical thickness for the Tian atlas). I decided to keep the function the same between the two packages for the sake of synchronicity.

Reply:
I'm happy to change the naming in both repos to something more informative. Maybe `cortex` and `no_cortex`?

Reply:
I'm happy with either as long as there is a note somewhere